src/topics/best-practices.md

```diff
@@ -56,27 +56,27 @@ all are required.
   known to work with this description under `SoftwareRequirement`.
 
 - `format` should be specified for all input and output `File`s.
-  Bioinformatics tools should use format identifiers from [EDAM][edam-example].
+  Bio-informatics tools should use format identifiers from [EDAM][edam-example].
```

Comment:
I think Bioinformatics is OK too. I see it more frequently than bio-informatics

Reply:
And just a few words down there's non-bioinformatics, not non-bio-informatics. For consistency I think bioinformatics would be better here.